packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -599,6 +600,9 @@ const MergeListOnBackspaceAfterList: BuildInEditFeature<PluginKeyboardEvent> = {
         return false;
     },
     handleEvent: (event, editor) => {
+        if (event.eventType == PluginEventType.KeyDown) {
+            event.handledByEditFeature = true;
```

Comment:
Instead of setting this flag here, you can set it in EditPlugin, when an edit feature is found and used, set this flag so the flag is common for all features.